tck/src/main/java/org/eclipse/microprofile/opentracing/tck/OpentracingClientTests.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2017 Contributors to the Eclipse Foundation
```

Comment:
Maybe you did this already, but I haven't seen an update to the NOTICE file to indicate the additional contributions by KevinG...  Any others?

Reply:
Updated